### PR TITLE
Modular PDA tweaks and bugfixes

### LINKED ISSUE #10439: AI IM tab has vanished
I play borg a lot and just noticed the AI IM tab is gone. Borgs can't use IMs now. Possibly connected the PDA update?

### PR DESCRIPTION
**Pull request by JohnWildkins**
changes:
  - rscadd: "Handheld modular computers now use flashlights as a light source. They may be turned on/off via a button in the PC header, and have their brightness adjusted in Hardware Configuration."
  - tweak: Message notifications should be much larger.
  - tweak: "Chat messages are now logged to the NTNet Monitoring program."
  - bugfix: "IDs are now unregistered from computers when the computer is destroyed."
  - bugfix: "Fix Eject Battery having a global view range."
  - bugfix: "Chat Clients are no longer set to 'silent' by default."
  - backend: "Added a new slider component for VueUIs."

Closes #10439

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!review

File: code/modules/modular_computers/computers/modular_computer/variables.dm [hunk 1]
```diff
@@ -47,7 +47,8 @@
 	var/message_output_range = 0							// Adds onto the output_message proc's range
 	var/max_hardware_size = 0								// Maximal hardware size. Currently, tablets have 1, laptops 2 and consoles 3. Limits what hardware types can be installed.
 	var/steel_sheet_cost = 5								// Amount of steel sheets refunded when disassembling an empty frame of this computer.
-	var/light_strength = 0									// Intensity of light this computer emits. Comparable to numbers light fixtures use.
+	light_range = 0									// Intensity of light this computer emits. Comparable to numbers light fixtures use.
+	light_power = 0
```
Comment (Amunak) [inline code/modules/modular_computers/computers/modular_computer/variables.dm:51]: I'd love to see a comment here saying what it's for and the valid range.

File: code/modules/modular_computers/computers/subtypes/dev_silicon.dm [hunk 2]
```diff
@@ -53,4 +53,30 @@
...
+/obj/item/modular_computer/silicon/proc/register_chat()
+	set_autorun("ntnrc_client")
+	enable_computer(null, TRUE) // passing null because we don't want the UI to open
+	minimize_program()
+
+/obj/item/modular_computer/silicon/verb/send_pda_message()
+	set category = "AI IM"
+	set name = "Send Direct Message"
+	set src in usr
+	if (usr.stat == 2)
```
Comment (Amunak) [inline code/modules/modular_computers/computers/subtypes/dev_silicon.dm:69]: I believe there are constants for these?

**Review by Amunak — APPROVED** [review]

File: code/modules/modular_computers/computers/subtypes/dev_silicon.dm (hunk 2 above)
Comment (Geevies) [inline code/modules/modular_computers/computers/subtypes/dev_silicon.dm:69]:
```suggestion
	if (usr.stat == DEAD
```

**Review by Geevies — APPROVED** [review]

**Comment by Geevies** [comment]
wait, im monkey

File: code/modules/modular_computers/computers/subtypes/dev_silicon.dm [hunk 3]
```diff
@@ -53,4 +53,29 @@
...
+/obj/item/modular_computer/silicon/proc/register_chat()
+	set_autorun("ntnrc_client")
+	enable_computer(null, TRUE) // passing null because we don't want the UI to open
+	minimize_program()
+
+/obj/item/modular_computer/silicon/verb/send_pda_message()
+	set category = "AI IM"
+	set name = "Send Direct Message"
+	set src in usr
+	if (usr.stat == DEAD
```
Comment (Geevies) [inline code/modules/modular_computers/computers/subtypes/dev_silicon.dm:69]:
```suggestion
	if (usr.stat == DEAD)
```

**Review by Amunak — APPROVED** [review]

**Pull request merged**